src/styles/base.tsx

```diff
@@ -0,0 +1,22 @@
+import { Dimensions } from 'react-native';
+
+export const dimensions = {
```

Comment:
This is a bit of difficult here because dimensions are available immediately, but they may change (e.g due to device rotation) so any rendering logic or styles that depend on these should try to call this function on every render. In addition the dimension only gives us only density independent layout size, meaning the value is not in px. If we want to have them in px we need to use PixelRatio eg: 
``` 
import { Dimensions, PixelRatio } from 'react-native';
const { width, height } = Dimensions.get('window');
PixelRatio.getPixelSizeForLayoutSize(width);
PixelRatio.getPixelSizeForLayoutSize(height);
```
If we don't want to define it globally we can also just use `PixelRatio.getPixelSizeForLayoutSize(Dimensions.get('window').width);`.